lab-anthony/test/greet-test.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
```

Comment:
Looks great, I really like that you tested what happens when the function gets different arguments.  Keep watching your edge cases! :D